docs/source/data-access-api/openapi.yml

```
@@ -0,0 +1,159 @@
+            `here <https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/naming-conventions.html#data-product-file-naming-conventions>`_.
+          required: true
+          schema:
+            type: string
+      responses:
+        '302':
+          description: Successful download
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                  format: uri
+        '400':
+          description: Unsuccessful download
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                  format: uri
+
+  '/query':
+    get:
+      tags:
+        - Query
+      summary: Query for file metadata
+      operationId: query
+      parameters:
+        - in: query
+          name: instrument
+          description: |
+            The instrument of interest (e.g. ``mag``). Supported instruments are listed
+            `here <https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/naming-conventions.html#data-product-file-naming-conventions>`_.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: data_level
+          description: |
+            The level of data product (e.g. ``l1a``). Supported levels are listed
+            `here <https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/naming-conventions.html#data-product-file-naming-conventions>`_.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: descriptor
+          description: |
+            The descriptor of interest (e.g. ``burst``). Supported descriptors are listed
+            `here <https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/naming-conventions.html#data-product-file-naming-conventions>`_.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: start_date
+          description: The start date of observation in the format ``YYYYMMDD``.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: end_date
+          description: The end date of observation in the format ``YYYYMMDD``.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: version
+          description: The version of data product in the format ``vXX-YY`` (e.g. ``v01-01``).
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: extension
+          description: |
+            The file extension of interest (e.g. ``pkts``), or ``*`` for all extensions. Supported extensions include
+            ``pkts`` and ``cdf``.
```

I think this is the only one left of "*", did you want to remove this too?
```suggestion
            The file extension of interest (e.g. ``pkts``). Supported extensions include
            ``pkts`` and ``cdf``.
```

---

Good catch!